astrowidgets/core.py

```diff
@@ -694,6 +694,13 @@ def add_markers(self, table, x_colname='x', y_colname='y',
             coord_x, coord_y = image.wcs.wcs.all_world2pix(coord_val.ra.deg,
                                                            coord_val.dec.deg,
                                                            0)
+            # In the event a *single* marker has been added, coord_x and coord_y
+            # will be scalars. Make them arrays always.
+            try:
+                len(coord_x)
+            except TypeError:
```

Comment:
`except` is computationally expensive if it is invoked a lot. Why not use `np.isscalar` as a check?

Reply:
Mostly because I forgot that `np.isscalar` existed...changing.